website/docs/enterprise/getting-started/install-enterprise-single.mdx

```diff
@@ -0,0 +1,1165 @@
+
+## Getting Started
+
+:::info Before Getting Started
+- [ ] You are familiar Weave GitOps Enterprise [Concepts](/docs/references/concepts/).
+- [ ] You have a **Git Repository** that you want to bootstrap with **write** permissions to push resources.
+- [ ] You have a Kubeconfig pointing to your **Management Cluster** with **cluster-admin** to create resources.
+- [ ] You have installed [Flux CLI](https://fluxcd.io/flux/installation/#install-the-flux-cli) and [Weave GitOps Enterprise CLI](#weave-gitops-enterprise-cli)
+- [ ] You have **Weave GitOps Enterprise Entitlements** provided by [Sales](/help-and-support/).
+:::
+
+:::tip Install Weave GitOps Enterprise CLI
+To do this, you can use either brew or curl:
+   <Tabs groupId="weave-gitops-cli" default="homebrew">
+   <TabItem value="homebrew" label="Homebrew">
+
+   ```bash
+    brew install weaveworks/tap/gitops-ee
+   ```
+
+   </TabItem>
+   <TabItem value="curl" label="curl">
+
+   ```bash
+    export VERSION=$VERSION
+    curl --silent --location "https://artifacts.wge.dev.weave.works/releases/bin/${VERSION}/gitops-$(uname)-$(uname -m).tar.gz" | tar xz -C /tmp
+    sudo mv /tmp/gitops /usr/local/bin
+    gitops version
+   ```
+
+   </TabItem>
+   </Tabs>
+:::
+
+Your Day 0 as is about getting Weave GitOps Enterprise up and running quickly. At glance, the workflow have the following stages:
+
+1. [Bootstrap Flux](#bootstrap-flux)
+2. [Entitlements](#entitlements)
+3. [Authentication](#authentication)
+4. [Authorization](#authorization)
+5. [Install WGE](#install-wge)
+6. [Access the dashboard](#access-the-dashboard)
+
+That you could execute in any of the following ways:
+
+- `CLI Interactive`: ideal if you have limited knowledge about the product and want guidance step-by-step.
+- `CLI Non-Interactive`: ideal if you are already familiar with Weave GitOps Enterprise and want to leverage the CLI for convenience.
+- `Manual`: ideal if you want to have full control on the process or your context cannot be handled via the CLI.
+
+:::warning Write Modes & Permissions
+Bootstrapping requires you to have both **cluster admin** permissions on the Management Cluster and **push** permissions
+to the Git repository to write the generated resources. In the case that you want just to generate the manifests but don't apply them,
+you could use Manual route or use the CLI with the `--export` flag.
+:::
+
+<Tabs groupId="flux bootstrap" default>
+<TabItem value="cli interactive" label="cli interactive">
+
+ <AlphaWarning/>
+
+ // TODO add a recording
+
+ ```bash
+ gitops bootstrap
+ ```
+
+</TabItem>
+<TabItem value="cli non interactive" label="cli non interactive">
+
+<AlphaWarning/>
+You could run the bootstrap command in non-interactive mode by providing the required configurations as flags. The following gives you an example to get started that you could adapt to your own context.
+
+
+// TODO add a recording
+
+```bash
+gitops bootstrap \
```

Comment:
Regarding non-interactive documentation we go for the following:
- Here we add the main complete examples that allows us to get to WGE up and running for day-0 (less aim for 2 - 3 examples max): 

- a) I have already flux bootstrapped
- b) I don't have flux bootstrapped  and want bootstrapping via ssh 

Any other cli tabs with configurations should only there when we are adding extra functionality to the manual version. clear example is odic that we have a sub-command.